src/view/filler.js

```diff
@@ -168,3 +164,39 @@ function jumpOverInlineFiller( evt, data ) {
 		}
 	}
 }
+
+// Checks if given node is a nbsp block filler.
+//
+// A &nbsp; is a block filler only if it is a single child of a block element.
+//
+// @param {Node} domNode DOM node.
+// @returns {Boolean}
+function isNbspFillerFiller( domNode ) {
+	const isNBSP = isText( domNode ) && domNode.data == '\u00A0';
+
+	return isNBSP && hasBlockParent( domNode ) && domNode.parentNode.childNodes.length === 1;
+}
+
+// Name of DOM nodes that are considered a block.
+const blockElements = [ 'p', 'div', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6' ];
```

Comment:
@Reinmar I don't like that this duplicates the code from a `DomConverter` - maybe we should intorduce `isBlockElement()` in ckeditor5-utisl/dom? That this could be unified there.

Reply:
We should use the same block elements array which is defined in `DomConverter`. The idea is that it can be extended by someone else if needed. So, we need to find a way to make `filler.js` us that property.

Reply:
One option would be to have a module here (in the engine) which exports a table.

This module would be imported by both `domconverter.js` and `filler.js`. The instance of this array would be shared by those modules. So, if you'd extend `DomConverter.blockElementNames` that'd also extend the value used by the `filler.js` module.

The downside of this would be that this would become a singleton. All editor instances would share this table. In fact, that'd mean that if you'd like to extend this array in your plugin's code (cause it adds some new element support to the editor), if you'd run 10 editors on one page, this element would be added 10 times. That's quite ugly.

So, I'd rather go in a different direction – e.g. move `isBlockFiller()` to `DomConverter`. Perhaps it doesn't make sense to have this as contextless util. Maybe it needs to be a part of bigger context.